how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,171 @@
+    * [Run Kata with rootless Podman](#run-kata-with-rootless-podman)
+    * [Appendix: Possible Errors](#appendix-possible-errors)
+        * [Error caused by agent or runtime version mismatch](#error-caused-by-agent-or-runtime-version-mismatch)
+        * [Missing registry file](#missing-registry-file)
+
+
+For an even more secure system, [Kata Containers](https://Katacontainers.io) can run workloads without a privileged user. Using [Podman](https://podman.io/) as the container engine, and [slirp4netns](https://github.com/rootless-containers/slirp4netns) for user-space networking.
+
+## Requirements
+- A Linux system, see [supported distributions](https://github.com/kata-containers/documentation/blob/master/install/README.md#supported-distributions) for an updated list.
+  - If using CentOS 7, `newuidmap` and `newgidmap` do not exist, and can be installed with:
+    ``` bash
+    $ (git clone https://github.com/shadow-maint/shadow; cd shadow; ./autogen.sh --prefix=/usr --enable-man; make && sudo make -C src install)
+    $ sudo bash -c 'echo 10000 > /proc/sys/user/max_user_namespaces'
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subuid"
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subgid"
+    ```
+- Golang 1.12
+
+## Installation
+Refer to the following table for the __minimum__ required software versions and the installation instructions:
+
+| Component       | Version | Install Instructions|
+| ----------------|:-------:|---------------------|
+| Podman          | WIP     | [see here](https://github.com/containers/libpod/blob/master/install.md)
+| slirp4netns     | 0.4.0   | [see here](https://github.com/rootless-containers/slirp4netns#quick-start)
+| Kata Containers | WIP     | [see here](https://github.com/kata-containers/documentation/blob/master/install/README.md)
+
+> **NOTE:**
+> If installing Podman with a package manager, there is usually no need to install slirp4netns separately.
+
+## Configuration
+Now that Kata Containers and Podman have been installed, they need to be configured for rootless execution.
+
+#### 1. Disable SELinux
+If SELinux is installed and enabled, it needs to be disabled with the following command (Kata Containers [does not support SELinux](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)).
+
+> **Warning:**
+> The following command may differ depending on the distro being used:
+
+```bash
+$ [ -f /etc/selinux/config ] && sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+#### 2. Add user to KVM group
+If running a KVM based hypervisor, the user running the workload needs to be added to the KVM group:
+```bash
+$ sudo usermod -a -G kvm $USER
+```
+
+#### 3. Reboot
+Reboot the system for the changes to take effect (a reboot is required when disabling SELinux, while logging out and back in is enough to have that user joining the `KVM` group).
```

Comment:
Suggested rewrite: 
... (when you disable SELinux you must reboot, while logging out and back in is enough to have that user joining the ‘KVM’ group).